packages/frontend/editor-ui/src/utils/formatters/dateFormatter.ts

```diff
@@ -25,4 +25,5 @@ export function convertToDisplayDate(fullDate: Date | string | number): {
 
 export const toDayMonth = (fullDate: Date | string) => dateformat(fullDate, 'd mmm');
 
-export const toTime = (fullDate: Date | string) => dateformat(fullDate, 'HH:MM:ss');
+export const toTime = (fullDate: Date | string, includeMillis: boolean = false) =>
+	dateformat(fullDate, includeMillis ? 'HH:MM:ss.l' : 'HH:MM:ss');
```

Comment:
```suggestion
export const toTime = (fullDate: Date | string, format:string = 'HH:MM:ss') =>
	dateformat(fullDate, format);
```

have you considered this approach?
it looks cleaner to me but it's a matter of taste, not blocking

Reply:
I think it's another sensible approach, but I prefer to keep it like this here (a) to avoid many inconsistent date formats in the app and (b) to not leak formatting details supported by the underlying library.